source/flux/parts/homescreen.js

```diff
@@ -2,20 +2,33 @@
 
 import {allViewNames as defaultViewOrder} from '../../views/views'
 import difference from 'lodash/difference'
-import {trackHomescreenOrder} from '../../analytics'
+import {
+	trackHomescreenOrder,
+	trackHomescreenDisabledItem,
+	trackHomescreenReenabledItem,
+} from '../../analytics'
 import * as storage from '../../lib/storage'
 import {type ReduxState} from '../index'
+import isEqual from 'lodash/isEqual'
+import debounce from 'lodash/debounce'
+
+// see https://css-tricks.com/debouncing-throttling-explained-examples/
+// we only invoke trackHomescreenOrder once every three seconds, to avoid
```

Comment:
Why invoke it on a timer when we can invoke it upon unmounting of the component? Wouldn’t it make sense to grab their final config instead?

Reply:
Because… because… 

My best answer is that I'm trying to isolate the side effects to the Redux action creators.

Reply:
Ok 👍